ads/adblade.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {writeScript, checkData, validateDataExists} from '../src/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function adblade(global, data) {
+  const adbladeFields = ['width', 'height', 'cid'];
+
+  checkData(data, adbladeFields);
+  validateDataExists(data, adbladeFields);
+
+  // create a data element so our script knows what to do
+  const ins = '<ins class="adbladeads" data-width="' + data.width
```

Comment:
My initial implementation did that, but I could not insert the element in the correct place, which led to my using "ins.outerHTML". I had to remove that because it broke the build. 

Any idea on how I can add the element object in the correct location? Is it just "global.document.getElementById('c').appendChild(i);"?